### PR TITLE
test(post): interlink integration tests

### PR DESCRIPTION
**Pull request by rofe**
While implementing requested changes to the interlink feature, I thought it would sure be nice to have integration tests 😉  so I looked into adding a simple framework which can easily be extended to IT other features.

### CODE REVIEW

File: test/integration/post.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      },
+      {
+        path: '/en/publish/2020/12/18/annual-end-of-year-predictions-cx-best-practices.html',
+        links: ['customer intelligence', 'customer profiles', 'customer experience', 'customer data', 'customer journey'],
+      },
+      {
+        path: '/en/publish/2020/10/23/optimize-email-deliverability-2020-holiday-season.html',
+        links: [],
+      }
+    ]
```
Comment (kptdobe) [inline test/integration/post.test.js:42]: I assume this is directly hitting production articles together with the interlink production config... 
If authors change the content or the keywords to link, the tests get broken... We should test on content we have control on.

**Review by kptdobe — CHANGES_REQUESTED** [review]
I am not sure this is a good idea as is: we must have control on the content, otherwise tests will start breaking without us doing anything. With one test, it could be fine but if we add a lot more, it will become a nightmare.

File: test/integration/post.test.js (hunk 1 above)
Comment (rofe) [reply test/integration/post.test.js:42]: You're right, I'll record them.

**Comment by rofe** [comment]
Added code intercepting requests to HTML pages and keywords.json and delivering static content. The JS code is executed on a local checkout.

The GH action doesn't finish even though the test succeeds, no clue why...

**Review by kptdobe — APPROVED** [review]

**Pull request merged**